non-protocol-specs/0012-NP-LIPE-liquidation-price-estimate.md

```diff
@@ -50,6 +50,14 @@ When including orders we sort the orders in the order they will get filled in (d
     - update $S^{\text{current}}$ to equal $S^{\text{order}}$,
   - otherwise return last calculated $S^{\text{liquidation}}$ (assumes other orders will get cancelled and the remaining position will be liquidated).
 
+### Margin mode considerations
+
+Depending on the [margining mode](../protocol/0019-MCAL-margin_calculator.md#margining-modes) selected by the party for the market on which its position is being considered the $\text{collateral available}$ will differ.
+
+Cross-margin mode: $\text{collateral available} = \text{margin account balance} + \text{general account balance}$.
+
+Isolated margin mode: $\text{collateral available} = \text{margin account balance} + \text{order margin account balance}$.
```

Comment:
@TomMcL / @Jiajia-Cui can you please check if this is correct for isolated margin? 

My impression was that to prop up a position in isolated mode it's only the margin account balance.

Reply:
That is right, so it should be 

isolated margin mode: $\text{collateral available} = \text{margin account balance}

Reply:
Agreed, order margin account shouldn't be touched until the actual positions have already been liquidated/will be

Reply:
Thanks, amended now.